docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -63,6 +65,8 @@ Before choosing your deployment option, complete these Google Cloud setup steps:
 
 1. Install the gcloud CLI, required by both deployment options. Follow the instructions provided by Google: [Install the gcloud CLI](https://cloud.google.com/sdk/docs/install).
 
+1. Enable egress rule. If your VPC infrastructure only allows egress to known endpoints, you will need to enable an egress rule to allow the operator to retrieve the certificates required for attestation. Follow the details on this document from Google to enable this: [VPC Service Controls](https://cloud.google.com/vpc-service-controls/docs/supported-products#table_confidential_space).
```

Comment:
Preferred to put the reason before the action:

To enable this, follow the details in this document from Google: [VPC Service Controls](https://cloud.google.com/vpc-service-controls/docs/supported-products#table_confidential_space).